backend/test/api/api_test.go

```
@@ -110,6 +111,84 @@ func TestStatusReturnsMetadata(t *testing.T) {
+	hold := make(chan struct{})
+	mock.StartHold = hold
+	server := newTestServerWithMock(t, mock)
+	defer server.Close()
+
+	var releaseOnce sync.Once
+	release := func() { releaseOnce.Do(func() { close(hold) }) }
+
+	startDone := make(chan struct{})
+	go func() {
+		defer close(startDone)
+		resp, err := http.Post(server.URL+"/v1/runtime/start", "application/json", nil)
```

golangci-lint (2.12.2)

[error] net/http.Post must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)